storage/innobase/srv/srv0srv.cc

```diff
@@ -2737,12 +2737,12 @@ void undo_rotate_default_master_key() {
 }
 
 bool srv_enable_redo_encryption(THD *thd) {
-  if (srv_redo_log_encrypt == REDO_LOG_ENCRYPT_MK) {
-    return srv_enable_redo_encryption_mk(thd);
-  }
-
-  if (srv_redo_log_encrypt == REDO_LOG_ENCRYPT_RK) {
-    return srv_enable_redo_encryption_rk(thd);
+  switch (srv_redo_log_encrypt) {
```

Comment:
Can you please check other usages of srv_redo_log_encrypt and convert them to switch case?

Reply:
I've checked, there are no other issues.